### PR TITLE
Add darc dependency for Microsoft.NET.Sdk.Web

### PR DESCRIPTION
**Pull request by joeloff**
- If this PR should not run tests please add text "skip[REMOVE_THIS]ci[REMOVE_THIS]please" (remove the marked text, no quotes).
- Please add description for changes you are making.
- If there is an issue related to this PR, please add the reference.

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -70,4 +70,8 @@
       https://dotnetfeed.blob.core.windows.net/orchestrated-release-2-1/20180725-02/final/index.json
     </RestoreSources>
   </PropertyGroup>
+  <!--Package names-->
+  <PropertyGroup>
```
Comment (joeloff) [inline eng/Versions.props]: This got created when I ran 'darc add'. Wasn't sure if this is needed.

**Comment by joeloff** [comment]
@livarcocc I already created a subscription

File: eng/Version.Details.xml [hunk 2]
```diff
@@ -41,6 +37,10 @@
       <Uri>https://github.com/microsoft/vstest</Uri>
       <Sha></Sha>
     </Dependency>
+    <Dependency Name="Microsoft.NET.Sdk.Web" Version="3.0.100-preview.18621.22">
+      <Uri>https://github.com/aspnet/websdk</Uri>
+      <Sha></Sha>
```
[the remark this answers is not in the document]
Comment (livarcocc) [reply eng/Version.Details.xml]: Yes. It will do that the first time it runs.

File: eng/Versions.props (hunk 1 above)
Comment (livarcocc) [reply eng/Versions.props]: This should not be needed. Can you remove?

cc @mmitche to confirm though.

File: eng/Version.Details.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (joeloff) [reply eng/Version.Details.xml]: ok, so did I miss a parameter? I ran **darc add --name 'Microsoft.NET.Sdk.Web' -type product --version 3.0.100-preview.18621.22 --repo https://github.com/aspnet/websdk**

Should I just manually add the commit SHA for now?

File: eng/Versions.props (hunk 1 above)
Comment (joeloff) [reply eng/Versions.props]: Will wait for @mmitche to confirm

Comment (mmitche) [reply eng/Versions.props]: It's not needed.  I plan on removing it.

File: eng/Version.Details.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (mmitche) [reply eng/Version.Details.xml]: You can leave it out.  Maestro will add it.

**Pull request merged**